### PR TITLE
[WFLY-18522] handle BootOperationFailedException if KernelServicesBui…

### PR DESCRIPTION
**Pull request by istudens**
…lder.build() fails

Issue: 
https://issues.redhat.com/browse/WFLY-18522
Depends on: 
https://issues.redhat.com/browse/WFCORE-6383 / https://github.com/wildfly/wildfly-core/pull/5659

### CODE REVIEW

File: picketlink/src/test/java/org/wildfly/extension/picketlink/subsystem/InvalidAttributeManagerDeclarationUnitTestCase.java [hunk 1]
```diff
@@ -22,10 +22,15 @@
 
 package org.wildfly.extension.picketlink.subsystem;
 
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.containsString;
 import static org.junit.Assert.assertFalse;
```
Comment (yersan) [inline picketlink/src/test/java/org/wildfly/extension/picketlink/subsystem/InvalidAttributeManagerDeclarationUnitTestCase.java:27]: Unused import

File: picketlink/src/test/java/org/wildfly/extension/picketlink/subsystem/RoleGeneratorDeclarationUnitTestCase.java [hunk 2]
```diff
@@ -22,10 +22,15 @@
 
 package org.wildfly.extension.picketlink.subsystem;
 
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.containsString;
 import static org.junit.Assert.assertFalse;
```
Comment (yersan) [inline picketlink/src/test/java/org/wildfly/extension/picketlink/subsystem/RoleGeneratorDeclarationUnitTestCase.java:27]: Unused import

**Review by yersan — CHANGES_REQUESTED** [review]

File: picketlink/src/test/java/org/wildfly/extension/picketlink/subsystem/RoleGeneratorDeclarationUnitTestCase.java (hunk 2 above)
Comment (istudens) [reply picketlink/src/test/java/org/wildfly/extension/picketlink/subsystem/RoleGeneratorDeclarationUnitTestCase.java]: oops, I am sorry I forgot to push the latest version

**Comment by yersan** [comment]
Incorporated in #17265

**Comment by bstansberry** [comment]
Thanks @istudens

**Pull request merged**